support/ipfs-crontab

```diff
@@ -1 +1 @@
-* */12 * * * /usr/local/bin/ipfs name publish $(cat /augur/ipfs-deploy/NEW_BUILD_HASH) > /var/log/cron.log
+* */24 * * * /usr/local/bin/ipfs name publish $(cat /augur/ipfs-deploy/NEW_BUILD_HASH) > /var/log/cron.log
```

Comment:
Curious why the move to 24 over 12?

Reply:
I read that using a non default key the default time-to-live is 12 hours, but saw on the API the default is 24 hours, so I changed. Also the publish can throw up a Warning that gets spit out as a ERROR and I wanted to minimize frequency of the message.

Reply:
Yeah, defaults to 24, but It’s actually configurable as well.  

I’d originally set it to 12 hours to just make sure there was never a gap.